sign/bdn/bdn.go

```diff
@@ -128,62 +122,67 @@ func (scheme *Scheme) Verify(x kyber.Point, msg, sig []byte) error {
 
 // AggregateSignatures aggregates the signatures using a coefficient for each
 // one of them where c = H(pk) and H: keyGroup -> R with R = {1, ..., 2^128}
-func (scheme *Scheme) AggregateSignatures(sigs [][]byte, mask *sign.Mask) (kyber.Point, error) {
-	if len(sigs) != mask.CountEnabled() {
-		return nil, errors.New("length of signatures and public keys must match")
-	}
-
-	coefs, err := hashPointToR(mask.Publics())
+func (scheme *Scheme) AggregateSignatures(sigs [][]byte, mask Mask) (kyber.Point, error) {
```

Comment:
We are in the process of preparing a V4 release anyway 👍

Reply:
In that case.... I can drop the interface and switch to a BDN specific mask if that works better for you (not sure how _much_ breaking you want).

Reply:
Yes, that would be totally fine by me. Actually Mask seems to only be used in `bdn` anyway, I'd be fine with it also being in internal or so, and have BDN expose the methods required to extract the list of participants  from an aggregate signature.

Summoning @pierluca  and @K1li4nL in case they have other opinions